Gem/Code/Source/Components/NetworkAiComponent.cpp

```diff
@@ -16,43 +16,14 @@
 
 namespace MultiplayerSample
 {
-    AZ_CVAR(bool, mps_botMode, false, nullptr, AZ::ConsoleFunctorFlags::Null, "If true, enable bot (AI) mode for client.");
```

Comment:
Not sure why this has been removed. Its required functionality for Network performance testing

Reply:
For some reason, this change is regurgitating changes already made in GameJam_2022_07. In this case, mps_botMode was moved elsewhere such that NetworkAiComponent is our long term home for server driven AI and the Movement Component handles "client as a bot" behavior where we have a connected client with artificially generated inputs.